tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
+                array(
+                    'AB' => true,
+                    'AC' => 'a',
+                ),
+                array(
+                    '@A' => true,
+                    '@C' => false,
+                ),
+            ),
+            '@A - @D' => array(
```

Comment:
in `@A` rule `AC=a`
in `@D` rule `AC=b`

`@A - @D` should still has `AC=a`, as nobody turn off this configuration and we allow one set to overwrite configuration of rule from different set